infrastructure/src/main/java/org/wso2/testgrid/infrastructure/aws/AWSManager.java

```diff
@@ -105,10 +106,11 @@ public Deployment createInfrastructure(Script script, String infraRepoDir) throw
 
         CreateStackRequest stackRequest = new CreateStackRequest();
         stackRequest.setStackName(cloudFormationName);
+        CloudFormationScriptPreprocessor cfScriptPreprocessor = new CloudFormationScriptPreprocessor();
```

Comment:
Can we create an instance variable for this?

Reply:
+1. Will move declaring to class level and initializing to constructor so that `createInfrastructure` will be using single processor object.